### PR TITLE
Make ORT REUSE-compliant

### PR DESCRIPTION
**Pull request by sschuberth**
See https://reuse.software/.

Use at least reuse-tool 0.6.0 [1] which excludes Git submodules by
default and execute "reuse lint".

[1] https://github.com/fsfe/reuse-tool/blob/master/CHANGELOG.md#060---2019-11-19

Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch-si.com>

### CODE REVIEW

**Comment by sschuberth** [comment]
@tsteenbe, this deprecates your `reuse-copyright` branch.

**Comment by sschuberth** [comment]
FYI, @mxmehl.

**Comment by mxmehl** [comment]
Thank you, @sschuberth! I see that you only added info in the dep5 file, so quite far away from the actual source files. Would it be possible to add copyright and license information directly in the file, at least for source code? The latest version of the reuse tool has a lot of nice capabilities to add headers (semi-)automatically.

**Comment by mxmehl** [comment]
While at it, you could add the REUSE check to the Travis CI: https://reuse.software/dev/#ci :)

**Comment by sschuberth** [comment]
> I see that you only added info in the dep5 file

Actually, no. All Kotlin source code files (`*.kt`) have headers which also get recognized by the `reuse` tool. I only added `dep5` entries for files that are

- special configuration files where a big headers reads awkward,
- documentation,
- third-party files that we want / need to take as-is,
- test assets,
- special resources.

But well, maybe for some of the Kotlin script files I could still add headers and get rid of their `dep5` entries. I'll look into that.

File: .reuse/dep5 [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Copyright: 2017-2020 HERE Europe B.V. <opensource@here.com>
+License: Apache-2.0
+
+Files: *.md
+Copyright: 2017-2020 HERE Europe B.V. <opensource@here.com>
+License: Apache-2.0
+
+Files: *.gradle.kts
+Copyright: 2017-2020 HERE Europe B.V. <opensource@here.com>
+License: Apache-2.0
```
Comment (mnonnenmacher) [inline .reuse/dep5]: I think you can remove this entry after #2102 is merged.

Comment (sschuberth) [reply .reuse/dep5]: Yes, that's was the intention behind that PR.

**Pull request merged**